_posts/blog/2021-04-21-jshell-accumulo-feature.md

```diff
@@ -0,0 +1,141 @@
+---
+Title: JShell Accumulo Feature
+Author: R. Dane Magbuhos
+Reviewers: Christopher Tubbs, Keith Turner
+---
+
+## Overview
+First introduced in Java 9+, [JShell][JShell Doc] is an interactive Read-Evaluate-Print-Loop (REPL) 
+Java tool that assess user's inputed declarations, statements, and expressions and outputs 
+the results. This tool provides a convenient way to test out and execute quick tasks with Accumulo
+in the terminal.
+
+## Major Features
+* During Accumulo build, produces a default JShell script called `jshell-init.jsh` 
+containing up-to-date [Accumulo Java APIs][public APIs] and [AccumuloClient][client] 
+build implementation
+
+* Startup JShell with default or custom JShell script 
+
+* Both JShell start up options automatically import all relevant Java APIs 
+
+## Booting Up JShell Accumulo
+After installing and configuring the [latest Accumulo update][accumulo-repo]
+follow the steps below to startup JShell:
+
+1) Open up a terminal and navigate to Accumulo's home directory 
+
+2) To startup JShell with **default script** use this command:
+
+```bash
+$ bin/accumulo jshell 
+```
+3) To startup JShell with **custom script** use this command:
+
+```bash
+$ bin/accumulo jshell --startup (file/path/to/custom_script.jsh)
+```
+## JShell Accumulo Default Script
+The auto-generated `jshell-init.jsh` is located in Accumulo's `conf/` directory. 
+Inside `jshell-init.jsh` contains [Accumulo Java APIs][public APIs] formatted as import statements 
+and [AccumuloClient][client] build implementation. On startup the script automatically loads in the 
+APIs and attempts to construct a client. Should additional APIs and/or code implementations be 
+needed, simply append them to `jshell-init.jsh`. Alternatively you can create a separate JShell 
+script and specify the custom script's file path on startup.
```

Comment:
Might be worth mentioning that you can append any options that you might append to the `jshell` command-line if you were running that directly.